symbolic-debuginfo/src/base.rs

```diff
@@ -56,6 +56,9 @@ pub enum ObjectKind {
     /// information corresponding to the original object file.
     Sources,
 
+    /// An Il2Cpp LineMapping json file.
+    Il2Cpp,
```

Comment:
this is maybe a bit a generic name for a very specific file format?

Reply:
We already use this naming in all other places, so it should be rather specific to what it relates to.